pkg/util/ppcpuusage/cpuusages.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2024 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ppcpuusage
+
+import (
+	"sync"
+	"time"
+)
+
+// CPUUsages records tidb/tikv cpu usages
+type CPUUsages struct {
+	TidbCPUTime time.Duration
+	TikvCPUTime time.Duration
+}
+
+// SQLCPUUsages is used to record sqlID and its cpu usages
+type SQLCPUUsages struct {
+	sync.Mutex
+	sqlID     uint64
+	cpuUsages CPUUsages
+}
```

Comment:
Why not use atomic.Int64?

Reply:
Since we need to check sqlID before update cpuUsages synchronously, atomic.Int64 can't ensure after sqlID check, the SQLCPUUsages has not been modified before update cpuUsages.